platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/cache/CookieCache.java

```diff
@@ -26,6 +28,8 @@
 import com.google.common.cache.RemovalListener;
 import com.google.common.cache.RemovalNotification;
 
```

Comment:
Validity check should be an operation on the SecurityAssertion interface or implementation. The CookieCache is at a different level of abstraction- it only understood Elements, not SecurityAssertions.